src/snyk/common/constants/errors.ts

```diff
@@ -0,0 +1,3 @@
+export const ERRORS = {
+  DOWNLOAD_FAILED: `Unable to download Snyk CLI. Possible causes: network issues, misconfigured release channel, or CLI version unavailability. To resolve: check connection, try a different release channel (e.g. preview), restart VS Code, or check Snyk GitHub for updates`,
```

Comment:
suggestion: `Unable to download the Snyk CLI. This could be caused by connectivity issues or the CLI not being available on the selected release channel.` I'd add a button "Details" or "Navigate to Troubleshooting Docs"